models/prices/prices_tokens.sql

```diff
@@ -156,7 +156,7 @@ VALUES
 
 
     ("1inch-1inch", "bnb", "1INCH", "0x111111111117dc0aa78b770fa6a738034120c302", 18),
-    ("aave-aave-token", "bnb", "AAVE", "0xfb6115445bff7b52feb98650c87f44907e58f802", 18),
+    ("aave-new", "bnb", "AAVE", "0xfb6115445bff7b52feb98650c87f44907e58f802", 18),
```

Comment:
However the address matches the address in the old api address for bnb chain.
https://api.coinpaprika.com/v1/coins/aave-aave-token

Reply:
oh i'll revert the bnb one then, I assumed they were all bad.